### ISSUE TITLE
more than one link between the same node is not possible

### ISSUE THREAD

**Issue by evemaria** [swimlane/ngx-graph#123]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
more than one link between the same node is not possible

**Expected behavior**
more than one link should be shown in between the same nodes with different labels

What is the motivation / use case for changing the behavior?
not able to modify the ngx-graph

ngx-graph version: - "^5.1.0",

@angular/cli: 1.2.7
node: 8.11.2

@angular/animations: 4.4.7
@angular/common: 4.4.7
@angular/compiler: 4.4.7
@angular/core: 4.4.7

Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Language: [all | TypeScript X.X | ES6/7 | ES5

**Comment by doorman02** [comment]
This would be useful feature. An idea instead of showing multiple arrows stacked labels above the same arrow could also work. But multiple arrows would also be nice. Not sure how to solve this in case of >10 links. For now I am just going to duplicate the target node.

**Comment by assafsun** [comment]
https://github.com/swimlane/ngx-graph/pull/159

**Comment by marjan-georgiev** [comment]
Released in 5.5.0.
